src/Configuration/ConfigurationFactory.php

```diff
@@ -150,6 +154,23 @@ public function create(
         );
     }
 
+    private static function retrieveCoverageBasePath(
+        ?string $existingCoveragePath,
+        string $tmpDir
+    ): string {
+        Assert::nullOrStringNotEmpty($existingCoveragePath);
+
+        if ($existingCoveragePath === null) {
+            return $tmpDir;
+        }
+
+        if (Path::isAbsolute($existingCoveragePath)) {
+            return $existingCoveragePath;
+        }
+
+        return sprintf('%s/%s', getcwd(), $existingCoveragePath);
```

Comment:
Not fond of the `getcwd()` here, should it be made relative to the config file? This would be a change, but IMO would make more sense

Reply:
sounds good to me, as it would be in line with other things that are related to the config file